src/backend/parser/parse_utilcmd.c

```diff
@@ -396,13 +396,19 @@ transformCreateStmt(CreateStmt *stmt, const char *queryString)
 													 likeDistributedBy, bQuiet);
 	}
 
+	/*
+	 * CBDB: for a foreign table, do not inherit source table's distribution policy.
```

Comment:
Seems that we can actually drop these lines. 

As far as i understand, #if 0 used to reduce rebase-time pains, but this code is greenplum originated, so should be OK to simply remove it

Reply:
Thanks for review.
> #if 0 used to reduce rebase-time pains,
 
Right.

> but this code is greenplum originated, so should be OK to simply remove it

There are two Upstream in CBDB, Postgres and GPDB. We are about 2 years late than GPDB7 and
 are working on cherry-pick from GPDB to catch up though it's archived.

`#if 0` is used to keep upstream codes context,  and avoid conflicts when cherry-pick.